webapp/i18n/en.json

```diff
@@ -818,6 +818,9 @@
   "admin.sql.keyTitle": "At Rest Encrypt Key:",
   "admin.sql.maxConnectionsDescription": "Maximum number of idle connections held open to the database.",
   "admin.sql.maxConnectionsExample": "E.g.: \"10\"",
+  "admin.sql.connectionTimeoutTitle": "Connection Timeout:",
+  "admin.sql.connectionTimeoutDescription": "The number of seconds a query can be executing for before being cancelled.",
```

Comment:
It's a timeout for each SQL query made to the database, if it takes longer than the connection timeout that specific query will be cancelled. For example a query like `SELECT * FROM Posts` will take a long time for a Posts table with millions of rows, if it took longer than the connection timeout then that query will be cancelled and return an error instead of blocking, causing a back-up of goroutines and saturating the DB